components/web/squid/Makefile

```diff
@@ -26,19 +26,19 @@
 #
 
 BUILD_BITS=             64
-USE_OPENSSL11=		yes
+OPENSSL_DEFAULT=	3.1
 include ../../../make-rules/shared-macros.mk
```

Comment:
3.1 is our default openssl. The variable to change this would be
OPENSSL_VERSION = 3.1